w03/d1/workshop/main.css

```diff
@@ -0,0 +1,12 @@
+h1 {
+text-align: center;
+
+}
+.mid{
+align-items: center;
+margin-top: 10px;
```

Comment:
Hi Mikias, 
My only feedback so far is to always double check your indentation.
Make sure the things that are in your css rules are indented one tab in.